src/modelbench/run.py

```diff
@@ -81,7 +70,7 @@ def cli() -> None:
     help="Path to directory containing custom branding.",
 )
 @click.option("--anonymize", type=int, help="Random number seed for consistent anonymization of SUTs")
-@click.option("--parallel", default=False, help="Experimentally run SUTs in parallel")
+@click.option("--parallel", default=False, help="Obsolete flag, soon to be removed")
```

Comment:
I was hoping Click provided a `deprecated` argument for options, but unfortunately it doesn't (natively). It does for commands. An enterprising soul added that for options here:

https://github.com/sqlfluff/sqlfluff/pull/3904/commits/a38512f217f911ffb575a2c3d54f8bbf3593dd95

Do we care?

Reply:
I looked for that argument as well. That particular code requires you to have an option you'd rather they use, which isn't the case here. There is an open issue for this: https://github.com/pallets/click/issues/2263

I've explained our use case, so perhaps one day they'll support it.

Reply:
I tried it out. If I do it without a specific warning, it prints this:

```
Version 0.5 of this benchmark is a proof of concept only. Results are not intended to indicate actual levels of AI system safety.

/home/william/projects/mlcommons/modelbench/src/modelbench/run.py:98: UserWarning: --parallel option is unnecessary; benchmarks are now always run in parallel
  warnings.warn("--parallel option is unnecessary; benchmarks are now always run in parallel")
```

And if I add DeprecationWarning to it it doesn't print anything at all. (That appears to be on purpose; I think you have to run `python` with -X dev to get it to show deprecation warnings.

Overall, I prefer the bare message.